### DISCUSSION TITLE
Right click to behave the same (Exiting) when an API menu opens than when any other Cadwork menus are open

### DISCUSSION THREAD

**Discussion by ancelin-charpentier** [cwapi3d/cwapi3dpython#213] [Q&A]
Hey there,
Many of my coworkers came back to me asking for "right click" to behave the same (Exiting) while having an API menu open as when any other Cadwork menu open.
At the moment right click does the exact same thing than a left click if you do it over a button.
But even more problematic, it also starts that button command if you right click while having your mouse further left on the screen which is very counter-intuitive for any Cadwork user. (See picture below)

![image](https://github.com/user-attachments/assets/21088193-7880-4edf-ac18-8d711a971c8a)

Is there a way for Cadwork to differentiate click types? 
tkinter does the differentiation but only if the pointer is over its frame which is no help.

I would appreciate any help re this.
Thanks guys

**Comment by ancelin-charpentier** [comment]
@Brunner246 , have seen the question above by any chance?

**Reply by Brunner246** [reply]
We will check if we can make the same behaviour available.
